images_config/ks-robot-master.cfg

```diff
@@ -0,0 +1,456 @@
+apt-get install nfs-kernel-server autofs -y -f -y
+mkdir /u
+
+##grep -q -F '/home /u none bind 0 0' /etc/fstab || sh -c '(echo "/home /u none bind 0 0") > /etc/fstab'
+##echo "/home /u none bind 0 0" >> /etc/fstab
+##mount /u
+
+grep -q -F '/u *(rw,fsid=0,sync,no_subtree_check)' /etc/exports || sh -c '(echo "/u *(rw,fsid=0,sync,no_subtree_check)") >> /etc/exports'
+echo "/u *(rw,fsid=0,sync,no_subtree_check)" >> /etc/exports
+sed -i 's/NEED_STADT\=/NEED_STADT\=yes/g' /etc/default/nfs-common
+if [ -d "/u/robot/git" ]
+then
+sed -i 's/home\/robot/u\/robot/g' /etc/passwd
+read -p "Reboot the computer for the changes to take effect (y/n)?" choice
+case "$choice" in 
+y|Y ) echo "Yes" && reboot;;
+n|N ) echo "No" && exit;;
+* ) echo "invalid"&& exit;;
+esac
+fi
+#############ssh-gedit-display ########################
+
+echo "X11Forwarding yes" >> /etc/ssh/sshd_config
+
+echo "X11UseLocalhost no" >> /etc/ssh/sshd_config
+echo "PermitRootLogin yes">> /etc/ssh/sshd_config
+/etc/init.d/ssh restart
+#######################################################
+##########################NTP CONFIGURATION############################
+HOST=$(cat /etc/hostname)
+echo $HOST
+
+echo "# Specify one or more NTP servers." >> /etc/ntp.conf
+echo "# Use servers from the NTP Pool Project. Approved by Ubuntu Technical Board" >> /etc/ntp.conf
+echo "# on 2011-02-08 (LP: #104525). See http://www.pool.ntp.org/join.html for" >> /etc/ntp.conf
+echo "# more information." >> /etc/ntp.conf
+echo "server 0.de.pool.ntp.org" >> /etc/ntp.conf
+echo "server 127.127.1.0 # local clock" >> /etc/ntp.conf
+echo "fudge 127.127.1.0 stratum 10" >> /etc/ntp.conf
+echo "restrict $HOST mask 255.255.255.0 nomodify notrap" >> /etc/ntp.conf
+echo "# By default, exchange time with everybody, but don't allow configuration." >> /etc/ntp.conf
+echo "restrict -4 default kod notrap nomodify nopeer noquery" >> /etc/ntp.conf
+echo "restrict -6 default kod notrap nomodify nopeer noquery" >> /etc/ntp.conf
+echo "# Local users may interrogate the ntp server more closely." >> /etc/ntp.conf
+echo "restrict 127.0.0.1" >> /etc/ntp.conf
+echo "restrict ::1" >> /etc/ntp.conf
+############################NETDATA TOOLS##############################
+
+apt-get install zlib1g-dev uuid-dev libmnl-dev gcc make git autoconf autoconf-archive autogen automake pkg-config curl -y
+git clone https://github.com/firehol/netdata.git --depth=1 /root/netdata
+cd /root/netdata
+./netdata-installer.sh
+############################NOMACHINE##################################
+
+apt-get install pulseaudio -y
+wget http://download.nomachine.com/download/5.1/Linux/nomachine_5.1.9_6_amd64.deb -P /root/nomachine_5.1.9_6_amd64.deb
+dpkg -i /root/nomachine_5.1.9_6_amd64.deb
+############################Candump tools##################################
+
+initctl stop candump2mongodb
+rm /etc/init/candump2mongodb.conf
+wget https://raw.githubusercontent.com/ipa320/setup_cob4/master/scripts/socket_buffer.py -P /usr/local/bin/socket_buffer.py
```

Comment:
for consistency:

`wget -O  /usr/local/bin/socket_buffer.py  https://raw.githubusercontent.com/ipa320/setup_cob4/master/scripts/socket_buffer.py`